resources/model/OpenStudio.idd

```diff
@@ -14186,6 +14186,347 @@ OS:HeatPump:PlantLoop:EIR:Cooling,
        \note quadratic curve = a + b*PLR + c*PLR**2 is typical, other univariate curves may be used
        \note PLR = part load ratio (cooling load/steady state capacity)
 
+OS:HeatPump:AirToWater:FuelFired:Heating,
+       \memo The object defines a fuel-fired absorption heat pump based on equation-fit models.
+       \min-fields 32
+   A1, \field Handle
+       \type handle
+       \required-field
+   A2, \field Name
+       \required-field
+       \reference HeatPumpAirToWaterFuelFiredHeatingNames
+       \note Name of the fuel fired absorption heat pump system system
+       \reference ConnectionObject
+   A3, \field Water Inlet Node Name
+       \type object-list
+       \required-field
+       \object-list ConnectionNames
+       \note Inlet node name of the water side connection
+   A4, \field Water Outlet Node Name
+       \type object-list
+       \required-field
+       \object-list ConnectionNames
+       \note Outlet node name of the water side connection
+   A5, \field Air Source Node Name
+       \type object-list
+       \object-list OutdoorAirNodeNames
+       \note This is the air source node name, which is the evaporator side of the heat pump in heating mode.
+       \note Enter the name of an OutdoorAir:Node object.
+   A6, \field Companion Cooling Heat Pump Name
+       \note The name of the companion HeatPump:AirToWater:FuelFired:Cooling object
+       \note This field is used for a heat pump with switchable heating and cooling mode.
+       \type object-list
+       \object-list HeatPumpAirToWaterFuelFiredCoolingNames
+   A7, \field Fuel Type
+       \required-field
+       \type choice
+       \key NaturalGas
+       \key Propane
+       \key FuelOilNo1
+       \key FuelOilNo2
+       \key Diesel
+       \key Gasoline
+       \key Coal
+       \key OtherFuel1
+       \key OtherFuel2
+       \note Fuel Type (NaturalGas, Propane, Gasoline, Diesel etc.)
+   A8, \field End-Use Subcategory
+       \type alpha
+       \retaincase
+       \default General
+       \note Any text may be used here to categorize the end-uses in the ABUPS End Uses by Subcategory table.
```

Comment:
I know we haven't been very consistent historically, but make that \required-field + hardcode to "General" in the Ctor.

(Doesn't matter but I do not like the position of this field, it should be the last one IMHO, to match most E+ objects. Anyways, keep it there to match E+...)

Reply:
Meh. I see the "default General". I don't care enough, let's go with that.